lib/jsdom/living/webstorage/Storage-impl.js

```diff
@@ -56,15 +56,15 @@ class StorageImpl {
       return;
     }
 
-    // Concatenate all keys and values to measure their size against the quota
-    let itemsConcat = key + value;
+    // Concatenate all keys and values to measure their length against the quota
+    let itemsTotalLength = key.length + value.length;
     for (const [curKey, curValue] of this._items) {
       // If the key already exists, skip it as it will be set to the new value instead
       if (key !== curKey) {
-        itemsConcat += curKey + curValue;
+        itemsTotalLength += curKey.length + curValue.length;
       }
     }
-    if (Buffer.byteLength(itemsConcat) > this._quota) {
+    if (itemsTotalLength > this._quota) {
       throw new DOMException(`The ${this._quota} byte storage quota has been exceeded.`, "QuotaExceededError");
```

Comment:
This DOMException references bytes as well.

Reply:
Great catch, thank you!!